src/Components/PluginZip.php

```diff
@@ -31,9 +31,17 @@ public function zip(string $directory, ?string $branch = null): string
         // Extract the git repository there
         $this->exec(sprintf('git archive %s | tar -x -C %s', escapeshellarg($branch), escapeshellarg($pluginName)));
 
-        // Install composer dependencies
-        if ($this->needComposerToRun($directory)) {
-            $this->exec('composer install --no-dev -n -o -d ' . escapeshellarg($pluginName));
+        $composerJson = $directory . '/' . $pluginName . '/composer.json';
+        $composerJsonBackup = $directory . '/' . $pluginName . '/composer.json';
+
+        if (file_exists($composerJson)) {
+            copy($composerJson, $composerJsonBackup);
```

Comment:
Are the both files not the same?

Reply:
😱 You are correct, I forgot the most important thing! Should be fixed now.